sdk/keyvault/keyvault-keys/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@azure/keyvault-keys",
   "sdk-type": "client",
   "author": "Microsoft Corporation",
-  "version": "4.0.0-preview.4",
+  "version": "4.0.0-preview.5",
```

Comment:
Are we changing the secrets version too?

Reply:
KeyVault-Secrets already has version 4.0.0-preview.5 in master 🌞

Reply:
The currently released version in NPM is: 4.0.0-preview.4, so we're fine, I believe.

Reply:
Oh, I see, it's already bumped